javamelody-core/src/main/java/net/bull/javamelody/Parameter.java

```diff
@@ -403,7 +403,17 @@
 	/**
 	 * <a href='https://www.datadog.com/'>Datadog</a> host for accessing API. 'api.datadoghq.com' is default.
 	 */
-	DATADOG_API_HOST("datadog-api-host");
+	DATADOG_API_HOST("datadog-api-host"),
+
+	/**
+	 * Integer value as minimum time collector for current requests
+	 */
+	CURRENT_REQUEST_MIN_DURATION_LOG("current-request-min-duration"),

```

Comment:
The name of the parameter should have a relation to the feature, which is about publishing metrics

Reply:
and it should include unit for a duration